default.nix

```diff
@@ -1,3 +1,11 @@
 { pkgs ? import <nixpkgs> {} }:
 
-pkgs.callPackage ./derivation.nix {}
+pkgs.callPackage ./derivation.nix {
+  withEmacs = true;
+    emacspkg = pkgs.emacsWithPackages
+      (epkgs: (with epkgs.melpaPackages; with epkgs.orgPackages; [
+      use-package
+      org-plus-contrib
+      htmlize
+    ]));
+}
```

Comment:
Prefer sane defaults in `derivation.nix`, see next comment for details.

Reply:
I'm not really sure on how to do it. 
```nix
, emacsWithPackages
, emacsPkg ? (emacsWithPackages (epkgs: (with epkgs.melpaPackages; with epkgs.orgPackages; [
      use-package
      org-plus-contrib
      htmlize
    ])))```

Like this? it seems ugly